src/amber/router/session/cookie_store.cr

```diff
@@ -28,19 +28,18 @@ module Amber::Router::Session
 
   # This is the default Cookie Store
   class CookieStore < AbstractStore
-    property secret : String
```

Comment:
It looks as though this property is never being used?

I'm pretty confused on how the `secret` is being used to sign the cookie.  We don't want to regenerate a new secret every time we restart the server or we will lose our session.  

Is that what this is currently doing?  If so, we will want to put back this property and rip out he automatic key generator.

Reply:
Yeah it's not but I my code is good now. I'll probably merge in the morning.